com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md

```diff
@@ -40,6 +40,7 @@ Path tracing uses the [Volume](Volumes.md) framework, so to enable this feature,
 | **Maximum Depth**           | Set the maximum number of light bounces in each path. You can not set this to be lower than Minimum Depth.<br /> **Note**: You can set this and Minimum Depth to 1 if you only want to direct lighting. You can set them both to 2 if you only want to visualize indirect lighting (which is only visible on the second bounce). |
 | **Maximum Intensity**       | Set a value to clamp the intensity of the light value each bounce returns. This avoids very bright, isolated pixels in the final result.<br />**Note**: This property can make the final image dimmer, so if the result looks dark, increase the value of this property. |
 | **Sky Importance Sampling** | Set the sky sampling mode. Importance sampling favors the brightest directions, which is beneficial when using a sky model with high contrast and very intense spots (like a sun, or street lights). On the other hand, it can be slightly detrimental when using a smooth, uniform sky. It is active by default for HDRI skies only, but can also be turned On and Off, regardless of the type of sky in use. |
+| **Denoising** | Set the denoising mode for the path tracer. This option is only available when the "Unity Denoiser Package" is installed in your project and the available options might change based on the version of this package. The denoising operation can be adjusted by the following parameters: <br> - **Use AOVs**: improves the detail retention after denoising. <br> - **Temporal**: improves temporal consistency when denoising frame sequences.|
```

Comment:
**Denoising** | Enable this feature to denoise the output of the the path tracer.  This feature is only available if the **Unity Denoiser Package** is installed in your project. The parameters available may differ depending on the package version. <br>
There are three denoising libraries available: <br>
- **Intel Open Image Denoise**<br>
- **NVIDIA OptiX**<br>
- **Radeon Image Filtering** (Experimental)<br>
 The default is **Intel Open Image Denoise**.<br>
You can also enable two additional settings:<br>
- **Use AOVs** (Arbitrary Output Variables): Enable this option to increase detail retention after denoising. <br> 
- **Temporal**: Enable this option to improve temporal consistency for denoised frame sequences.<br> 

-----------------
I thought the Denoiser Package was planned to be included with Unity by default, as a core package? That doesn't mean you shouldn't mention that it needs to be installed, just wanted to check this. Also, later we should add a link to the docs for the denoiser package here.

Reply:
Altough, there's no Radeon filter anymore in the latest change so I think it has been removed for good.

Reply:
@ValGrimm-U3D I have rephrased this a bit, because as Remi said we expose only the two backends in HDRP (for now) and there is also the None option in the dropdown (which is the default). Can you please check again? thanks.